core/src/filed/backup.cc

```diff
@@ -1182,7 +1174,7 @@ bool EncodeAndSendAttributes(JobControlRecord* jcr,
 
   Dmsg1(300, "encode_and_send_attrs fname=%s\n", ff_pkt->fname);
   /** Find what data stream we will use, then encode the attributes */
-  if ((data_stream = SelectDataStream(ff_pkt, me->compatible)) == STREAM_NONE) {
+  if ((data_stream = SelectDataStream(ff_pkt)) == STREAM_NONE) {
```

Comment:
The compatible will not be as easy to remove as I thought, so we need to keep it for now.